tests/cases/fourslash/extract-method41.ts

```diff
@@ -17,4 +17,4 @@
 
 goTo.select("start", "end");
 verify.refactorAvailable("Extract Symbol", "function_scope_1");
-verify.not.refactorAvailable("Extract Symbol", "function_scope_2");
+verify.refactorAvailable("Extract Symbol", "function_scope_2");
```

Comment:
This is not the desired behaviour. We don't want to allow extracting this selection to a function in _module scope_ because it contains a reference to `this`. I'm not sure how to prevent this.